project/generate-dockerfile.sh

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+set -e
+
```

Comment:
We should probably add a comment in here so that people know that the default is x86 and that if $ARCH is set (even to x86) then it'll assume we want to use gcc-go.  I have the correct, don't I?  Or said another way, I don't think you specify $ARCH to any value (if you wanted to be really anal) and get the default behavior.